### ISSUE TITLE
Comma character prints incorrectly

### ISSUE THREAD

**Issue by lverns** [bpsm/edn-java#45]
```java
import us.bpsm.edn.printer.Printers;

public class CommaBug {

    public static void main(String[] args){
        System.out.println(Printers.printString(','));
    }
}
```
Running this program produces the following output
```
Exception in thread "main" us.bpsm.edn.EdnException: Whitespace character 0x2c is unsupported.
        at us.bpsm.edn.printer.Printers$11.eval(Printers.java:384)
        at us.bpsm.edn.printer.Printers$11.eval(Printers.java:357)
        at us.bpsm.edn.printer.Printers$1.printValue(Printers.java:142)
        at us.bpsm.edn.printer.Printers.printString(Printers.java:74)
        at us.bpsm.edn.printer.Printers.printString(Printers.java:56)
```
The expected output is
```
\,
```

**Comment by lverns** [comment]
Note that the *parser* handles commas correctly. 
```java
import us.bpsm.edn.parser.*;
import java.io.*;

public class CommaGood {

    public static void main(String[] args){
        Parseable pbr = Parsers.newParseable("\\,");
        Parser p = Parsers.newParser(Parsers.defaultConfiguration());
        char val = (char) p.nextValue(pbr);

        System.out.println(val == ',');
    }
}
```
This produces `true` as expected.

**Comment by bpsm** [comment]
Thanks very much for the bug report. I'll get a fix out this week.

**Comment by bpsm** [comment]
Fixed with release 0.4.7
